// Copyright 2023 Google LLC. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Messages for configuring component generated alerts.

syntax = "proto2";

package tfx.orchestration.experimental.core;

message ComponentGeneratedAlertInfo {
  optional string alert_name = 1;
  optional string alert_body = 2;
}

message ComponentGeneratedAlertList {
  repeated ComponentGeneratedAlertInfo component_generated_alert_list = 1;
}
